corehq/apps/repeaters/models.py

```diff
@@ -608,10 +608,13 @@ def _fail(self, reason, response):
             self.last_checked = datetime.utcnow()
             self.cancel()
         self.failure_reason = reason
-        log_counter(REPEATER_ERROR_COUNT, {
-            '_id': self._id,
-            'reason': reason,
-            'target_url': self.url,
+        datadog_counter(REPEATER_ERROR_COUNT, {
```

Comment:
Curious if there will be lot of these or if there are any datadog limits/throttling